collector/filesystem_bsd.go

```diff
@@ -33,6 +33,7 @@ import "C"
 
 const (
 	defIgnoredMountPoints = "^/(dev)($|/)"
+	MNT_RDONLY            = 0x1
```

Comment:
:+1: for keeping it Go-style, especially since the semantics change when the variable starts upper-cased.